### PR TITLE
add stale issue check workflow

### PR DESCRIPTION
**Pull request by h1dden-da3m0n**
### Description

This PR aims to add a GitHub Action workflow to monitor, label and close stale issues.
It uses the Jellyfin-Bot token to comment and close stale Issues.

> NOTE: It explicitly does not touch PRs

### Changes

* add a basic GitHub action workflow to monitor, label, and close stale issues within this repository

### Issues

* successor to the discontinued Stale-Bot config

### CODE REVIEW

File: .github/workflows/repo-stale.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
```
Comment (nielsvanvelzen) [inline .github/workflows/repo-stale.yaml:14]: The workflows in this repository use explicit versions, so this should be `ubuntu-20.04`.

File: .github/workflows/repo-stale.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Issue Stale Check
```
Comment (nielsvanvelzen) [inline .github/workflows/repo-stale.yaml:1]:
```suggestion
name: Issue stale check
```
For consistency with the "Merge conflict labeler" workflow.

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

**Review by Maxr1998 — APPROVED** [review]
I fear some good issues could get closed by this since we're not especially fast with fixing them, but you can always bump them, I guess. Should help us direct our focus too.
But generally, this workflow looks good to me.

**Comment by h1dden-da3m0n** [comment]
> I fear some good issues could get closed by this since we're not especially fast with fixing them, but you can always bump them, I guess. Should help us direct our focus too. But generally, this workflow looks good to me.

as long as you label them `enhancement` or `confirmed` issues will get skipped. You can always also add more to the list of issues to exempt from ever getting stale ;)

**Review by nielsvanvelzen — APPROVED** [review]

**Comment by Maxr1998** [comment]
That's good to know. Gotta add a confirmed label then.

**Pull request merged**